metadoc-js/src/main/scala/monaco/Monaco.scala

```diff
@@ -1962,6 +2017,7 @@ package editor {
         diffEditor: IStandaloneDiffEditor,
         opts: IDiffNavigatorOptions = ???
     ): IDiffNavigator = js.native
+    // NOTE: Do not call this method, use MetadocTextModelService.createModel instead.
```

Comment:
Amongst other things, it will also be useful to have a single place to create models in #34